email/email.go

```diff
@@ -171,6 +173,10 @@ func makeVerificationUrl(secretUUID uuid.UUID) string {
 	return fmt.Sprintf("https://api.fluidkeys.com/v1/email/verify/%s", secretUUID.String())
 }
 
+func makeDevVerificationUrl(secretUUID uuid.UUID) string {
+	return fmt.Sprintf("http://localhost:4747/v1/email/verify/%s", secretUUID.String())
```

Comment:
even neater would be to generate this from the host / port in the request, rather than hard code it!